docs/enterprise-setup/implementation-guide.md

```diff
@@ -81,11 +81,162 @@ To configure basic auth (deploy without SSO), remove the entire `auth:` section
 
 </details>
 
+#### Configuring the Airbyte Database
+
+For Self-Managed Enterprise deployments, we advise against using the default Postgres database (`airbyte/db`) that Airbyte spins up within the Kubernetes cluster. For production, you should instead use a dedicated instance for better reliability, and backups (such as AWS RDS or GCP Cloud SQL).
+
+We assume in the following that you've already configured a Postgres instance:
+
+1. In the `charts/airbyte/values.yaml` file, disable the default Postgres database (`airbyte/db`):
+
+```yaml
+postgresql:
+    enabled: false
+```
+
+2. In the `charts/airbyte/values.yaml` file, enable and configure the external Postgres database:
+
+```yaml
+externalDatabase:   
+    host: ## Database host
+    user: ## Non-root username for the Airbyte database
+    database: db-airbyte ## Database name
+    port: 5432 ## Database port number 
+```
+
+For the non-root user's password which has database access, you may use `password`, `existingSecret` or `jdbcUrl`. We recommend using `existingSecret`, or injecting sensitive fields from your own external secret store. Each of these parameters is mutually exclusive: 
+
+```yaml
+externalDatabase:
+    ...
+    password: ## Password for non-root database user
+    existingSecret: ## The name of an existing Kubernetes secret containing the password.
+    existingSecretPasswordKey: ## The Kubernetes secret key containing the password.
+    jdbcUrl: "jdbc:postgresql://<user>:<password>@localhost:5432/db-airbyte" ## Full database JDBC URL. You can also add additional arguments.
+```
+
+The optional `jdbcUrl` field should be entered in the following format: `jdbc:postgresql://localhost:5432/db-airbyte`. We recommend against using this unless you need to add additional extra arguments can be passed to the JDBC driver at this time (e.g. to handle SSL).
+
+3. Finally, add this configuration into the `global` section of the `charts/airbyte/values.yaml` as well:
+
+```yaml
+global:
+  ...
+  database:
+    secretName: "airbyte-enterprise-airbyte-secrets" ## The name of the existing Kubernetes secret entered above.
+    secretValue: "DATABASE_PASSWORD" ## The Kubernetes secret key entered above.
+    host: "localhost" ## Database host entered above.
+    port: "5432" ## Database port entered above.
+```
+
+If you've used `password` or `jdbcUrl` above instead using a pre-existing Kubernetes secret, then your `secretName` will be `{release-name}-airbyte-secrets`, and `secretValue` will be `DATABASE_PASSWORD`.
+
+#### Configuring External Logging
```

Comment:
Don't we also need external state storage configured as well.

Reply:
This is where things seem to fall apart for me. Right now it sort of reads like the state configuration is coupled to the configuration of logging, as it is the same credentials that you would presumably be using for both, but you can certainly use one product for state and another for logging.

That said, we prob to need an explicit section for configuration of state storage

Reply:
I'm happy to add a section for state storage once it's clear what needs to be configured :)